create_pdfs.py

```diff
@@ -37,8 +37,9 @@ def parse_args(args=sys.argv[1:]):
     parser.add_argument('-i', '--issued-date', help='optional issue date')
     parser.add_argument(
         '-T',
```

Comment:
I too would prefer if argument descriptions matched their argument flags but they don't always match.  It's fact of life.   We shouldn't arbitrarily change arguments since there is no real reward for the risk.